hack/ci/run_e2e

```diff
@@ -40,6 +39,11 @@ if [[ ${BUILD_E2E} == "true" ]]; then
   TEST_IMAGE=${TEST_IMAGE} hack/build/e2e/docker_push
 fi
 
+if [ -z "${PASSES-}" ]; then
+  echo "No PASSES specified. Skipping tests"
```

Comment:
Might want to use tab instead?

Reply:
I can fix this and other indents in a follow-up PR. Testing the master jenkins job right now with `PASSES=""`

Reply:
No worry about it.
Just remind that when we have PR for jenkins scripts.